src/io/_Private/GenericException.php

```diff
@@ -0,0 +1,17 @@
+<?hh
+/*
+ *  Copyright (c) 2004-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace HH\Lib\Experimental\IO\_Private;
+
+use namespace HH\Lib\Experimental\{IO, OS};
+
+final class GenericException extends IO\Exception implements IO\ExceptionWithErrno {
```

Comment:
-1, i think developers should have the ability to catch a particulate exception, without having to catch all `IO\Exception`s ( in order to catch `_Private\GenericException` ), exception name should mean something. and developers should be able to understand what's the issue from the exception name.